### DISCUSSION TITLE
How to use it inside Laravel Sail ?

### DISCUSSION THREAD

**Discussion by AlexisVS** [orchestral/testbench-dusk#82] [Q&A]
Hello everyone,

I hope you're doing well. This is my first time using orchestra/testbench-dusk, and I'm facing an issue with the package's default configuration in my Laravel Sail environment.

In my setup, I'm not using the default environment provided by Laravel. Instead, I rely on Laravel Sail as my environment for running my application. However, the package currently assumes the usage of php artisan serve on localhost:8001, which doesn't align with my configuration.

As a result, I'm seeking guidance on how to adjust the package to work seamlessly within my Laravel Sail environment. Are there any specific configuration changes or alternatives I should consider to ensure compatibility?

Thank you all for your assistance and understanding as I navigate through this process.

`DuskTestCase.php`

```PHP
<?php

namespace AlexisVS\MultipassTestingModule\Tests;

use AlexisVS\MultipassTestingModule\MultipassTestingModuleServiceProvider;
use Orchestra\Testbench\Dusk\TestCase as OrchestraDuskTestCase;

class DuskTestCase extends OrchestraDuskTestCase
{
    /**
     * Automatically enables package discoveries.
     */
    protected $enablesPackageDiscoveries = true;

    /**
     * Automatically loads environment variables.
     */
    protected $loadEnvironmentVariables = true;

    public function getEnvironmentSetUp($app): void
    {
        config()->set('database.default', 'testing');
    }



    protected function getPackageProviders($app): array
    {
        return [
            MultipassTestingModuleServiceProvider::class,
        ];
    }

    public static function applicationBasePath(): string
    {
        return __DIR__ . '/../../../../';
    }

    protected function getBasePath(): string
    {
        return __DIR__ . '/../../../../';
    }
}

```

**Comment by Keoghan** [comment]
Hi Alexis, 

I haven't used sail, but you will need to set up a chromium container to use (see Sail docs - https://laravel.com/docs/10.x/sail#laravel-dusk). 

Once you have done this, you will need to override the settings for the driver, similar to the docs for Dusk (see Dusk docs - https://laravel.com/docs/10.x/dusk#using-other-browsers).

Hope that helps.

Keoghan
